front/app/components/UI/MentionsTextArea/index.tsx

```diff
@@ -232,6 +237,13 @@ const MentionsTextArea = ({
         }));
       }
 
+      if (showUniqueUsers && value) {
+        const ids = extractIdsFromValue(value);
+        const uniqueUsers = users.filter((user) => !ids.includes(user.id));
```

Comment:
Not sure if it's the case - but is there any chance this would result in a slow UX if there are many users on the platform?